docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```diff
@@ -0,0 +1,459 @@
+# Microsoft.Diagnostics.NETCore.Client
+
+**This article applies to: ✔️** .NET Core 3.0 SDK and later versions
```

Comment:
Should this message apply to the version of apps the library can _target_ or the version of apps the library can _run_ on?